mammoth/docx/body_xml.py

```diff
@@ -71,18 +71,26 @@ def _create_reader(numbering, content_types, relationships, styles, docx_file, f
 
     def text(element):
         return _success(documents.Text(_inner_text(element)))
-
+   
+    def element_or_value(element):
+        if element and ("w:val" in element.attributes):
+            element_value = element.attributes["w:val"]
+            if element_value == 'false':
+                return False
+            elif element_value == 'true':
```

Comment:
As a minor stylistic point: Mammoth tends to use double quotes instead of single quotes.